### DISCUSSION TITLE
Issue realeted adding more than 1 gb files into git

### DISCUSSION THREAD

**Discussion by Deepaky19** [community/community#70609] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Hello team i am facing this error while pushing file in branch.
I added LFS also for large files

while sending more then 100mb file via GIT LFS to github repo, we encounter an error 

 "batch response: This repository is over its data quota. Account responsible for LFS bandwidth should purchase more data packs to restore access."

**Comment by markdennis122** [comment]
Should large binary files be in the repository, my opinion is no.

But if you really need version control on binaries or assets consider something like git-fat, or git-lfs

https://github.com/jedbrown/git-fat

https://git-lfs.github.com/

**Comment by markdennis122** [comment]
The idea of Git is to track small files with code. So the suggestion is to include your source files and ignore very big files like static files.

if you include a 4GB file every people who clone the repo have to clone the whole 4GB. Thats not the right way.

**Comment by Deepaky19** [comment]
It's working Thanks
